_data/other.yml

```diff
@@ -197,6 +197,13 @@ websites:
       software: Yes
       doc: https://www.findmyshift.com/help/what-is-two-factor-authentication
 
+    - name: Fiverr
+      url: https://www.fiverr.com/
+      img: fiverr.png
+      tfa: No
+      twitter: fiverr
```

Comment:
As with https://github.com/2factorauth/twofactorauth/pull/2797#discussion_r142583002, should [`fiverr`](https://twitter.com/fiverr) be changed to [`FiverrSupport`](https://twitter.com/FiverrSupport)?

Reply:
It seems that their support handle actively answers questions so that one would probably be best.